corehq/apps/reports/commtrack/data_sources.py

```diff
@@ -461,6 +461,7 @@ def get_data(self):
             )
 
             form_xmlnses = [form['xmlns'] for form in self.all_relevant_forms.values()]
+            form_xmlnses.append(COMMTRACK_REPORT_XMLNS)
```

Comment:
yeah fair point, but also agree probably an edge case and not worth the effort to do this fix properly